### ISSUE TITLE
Gallery block needs to be resolved after first time save

### ISSUE THREAD

**Issue by SkilletCreek** [WordPress/gutenberg#14231]
**Describe the bug**
Very similar to this report, but applies to larger galleries. https://github.com/WordPress/gutenberg/issues/10515

To Reproduce

- Create a page or post with Gutenberg and add a gallery. (In this case, there are about 50 images. I did not get this error for more normal gallery sites, say under 10 images.)
- Save the page and view the page.
- Edit the page.
- Now the image block will give the 'This block contains unexpected or invalid content.' error. 
Resolving will not fix the problem. Converting to HTML breaks the layout. 

Expected behavior
Block should be still be a gallery block when you return to edit the page and not display this error.

**Screenshots**

<img width="712" alt="screen shot 2019-03-05 at 2 42 50 am" src="https://user-images.githubusercontent.com/48239983/53792060-6fc89400-3ef0-11e9-85cc-e884365c5a6d.png">

<img width="856" alt="screen shot 2019-03-05 at 2 44 35 am" src="https://user-images.githubusercontent.com/48239983/53792157-a56d7d00-3ef0-11e9-9a0b-6dcb5f1234d5.png">

**Desktop (please complete the following information):**
 - OS: Mac[10.14.3]
 - Browser [Safari]
 - Version [12.0.3]

**Additional context**
- Gutenberg 5.1

**Comment by youknowriad** [comment]
I'm not able to reproduce this error, do you have plugins tweaking the core gallery block? any other particularity?

**Comment by SkilletCreek** [comment]
Thanks for the response. The only plugins that I have which I think may relate to this are Simple Light Box, Atomic Blocks, Stackable & Ultimate Addons (I'm trying these different builders.) But I'm not directly using any of these on the galleries. 

With a bit of testing, I found that I was able go up to around a 40 image gallery without an issue. Somewhere past that it breaks (Once you save, then return and try to edit it). So I tried putting 2 galleries on the same page. One 40 images, then right below another 20 images. Then after saving, exiting and returning, the second one is broken. 

The only workaround I found was simply to use the Classic editor under "Formatting" to build the galleries. That works fine. It's not as pretty but at least I can save it, then return and edit it. :)
